gno.land/cmd/gnoland/main.go

```diff
@@ -314,3 +309,103 @@ func loadGenesisBalances(path string) []string {
 	}
 	return balances
 }
+
+// listGnoPkgs lists all gno packages in the given root directory.
+func listGnoPkgs(root string) ([]pkg, error) {
+	var pkgs []pkg
+
+	err := filepath.WalkDir(root, func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if d.IsDir() {
```

Comment:
you can do the opposite and remove one indentation
```suggestion
		if !d.IsDir() {
		    return nil
		}
```

Reply:
Resolved-by: a3e978ad739086056b92d14ddb5f27a66df4e14c